File: test_ton_dest.py
import urllib.request
import json
url = 'https://tonapi.io/v2/accounts/UQBTW0bj2lR0NQ2WdlRVyuLNNO55uokEo9hauUMWOArxAKVx/jettons/history?limit=10'
try:
    req = urllib.request.Request(url, headers={'Accept': 'application/json'})
    with urllib.request.urlopen(req) as response:
        print(response.read().decode('utf-8'))
except Exception as e:
    print('Error:', e)

File: test_ton_dest.js
const dest = 'UQBTW0bj2lR0NQ2WdlRVyuLNNO55uokEo9hauUMWOArxAKVx';
fetch(`https://tonapi.io/v2/accounts/${dest}/jettons/history?limit=10`, {
    headers: { 'Accept': 'application/json' }
}).then(async res => {
    console.log("Status:", res.status);
    const text = await res.text();
    console.log("Raw Response:");
    console.log(text);
}).catch(console.error);
